package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "@okta/oktasdk-java",
+  "version": "0.0.1",
```

Comment:
Not sure if we care about this version number matching the pom.xml. I don't think it matters, and it just adds to what we have to think about when we bump the version of the artifact.

Also curious if we should do `private: true` to prevent accidental pushes to npm